src/x/debug/config/config.go

```diff
@@ -18,31 +18,19 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package pprof
+package config
 
-import (
-	"net/http"
-	"net/http/httptest"
-	"testing"
+import "runtime"
 
-	"github.com/stretchr/testify/require"
-)
+// DebugConfiguration for the debug package.
+type DebugConfiguration struct {
 
-func TestHandler(t *testing.T) {
-	s := httptest.NewServer(handler())
-	url := s.URL + pprofPath
-	defer s.Close()
+	// MutexProfileFraction is used to set the runtime.SetMutexProfileFraction to report mutex convention events.
+	// See https://tip.golang.org/pkg/runtime/#SetMutexProfileFraction for more details about the values.
+	MutexProfileFraction int `yaml:"mutex_profile_fraction"`
```

Comment:
all our config yaml properties are camel case, probably better for consistency for this to be mutexProfileFraction?

Reply:
ah k